lua/nordic/groups/native.lua

```diff
@@ -74,8 +74,8 @@ function M.get_groups()
     G.markdownHeadingDelimiter = { link = 'mkdHeading' }
     G.markdownCode = { link = 'CodeBlock' }
     G.markdownCodeBlock = { link = 'CodeBlock' }
-    G.markdownH1 = { link = 'htmlH1' }
-    G.markdownH2 = { link = 'htmlH2' }
+    G.markdownH1 = { fg = C.yellow.base }
+    G.markdownH2 = { fg = C.orange.base }
```

Comment:
I don't know what this does could you explain it?

Reply:
Unfortunately, to get the headings to work, both the `@markup.heading.x` groups and the `markdownHx` groups had to be changed in order to get the desired settings to work. I unlinked those because it would make `H1-2` bold, but if we decide to make `H1-6` bold, we can add that link back